runtime/vm/stringhelpers.cpp

```diff
@@ -323,8 +322,8 @@ copyStringToUTF8WithMemAlloc(J9VMThread *vmThread, j9object_t string, UDATA stri
 	Assert_VM_notNull(string);
 
 	U_8* result = NULL;
-	UDATA stringLength = J9VMJAVALANGSTRING_LENGTH(vmThread, string) * 3;
-	UDATA length = stringLength + prependStrLength;
+	UDATA stringLength = J9VMJAVALANGSTRING_LENGTH(vmThread, string);
+	UDATA length = prependStrLength + stringLength * 3;
```

Comment:
For clarity, please bracket `stringLength * 3`

Reply:
Fixed in b67c08c